docs/usage.md

```diff
@@ -1,117 +1,179 @@
+nextflow run nf-core/methylseq --reads '*_R{1,2}.fastq.gz' -profile docker
 ```
 
-Note that the pipeline will create files in your working directory:
+This will launch the pipeline with the `docker` configuration profile. See below for more information about profiles.
+
+Note that the pipeline will create the following files in your working directory:
 
 ```bash
-work/           # Directory containing the nextflow working files
-results/        # Finished results (configurable, see below)
+work            # Directory containing the nextflow working files
+results         # Finished results (configurable, see below)
 .nextflow_log   # Log file from Nextflow
-.nextflow/      # Nextflow cache and history information
+# Other nextflow hidden files, eg. history of pipeline runs and old logs.
 ```
 
+### Updating the pipeline
 
-## Inputs and outputs
-
-### `--reads`
-Location of the input FastQ files:
+When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure that you're running the latest version of the pipeline, make sure that you regularly update the cached version of the pipeline:
 
 ```bash
- --reads 'path/to/data/sample_*_{1,2}.fastq'
+nextflow pull nf-core/methylseq
 ```
 
-**NB: Must be enclosed in quotes!**
+### Reproducibility
 
-Note that the `{1,2}` parentheses are required to specify paired end data. The file path should be in quotation marks to prevent shell glob expansion.
+It's a good idea to specify a pipeline version when running the pipeline on your data. This ensures that a specific version of the pipeline code and software are used when you run your pipeline. If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.
 
-If left unspecified, the pipeline will assume that the data is in a directory called `data` in the working directory (`data/*{1,2}.fastq.gz`).
+First, go to the [nf-core/methylseq releases page](https://github.com/nf-core/methylseq/releases) and find the latest version number - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`.
 
-### `--singleEnd`
-By default, the pipeline expects paired-end data. If you have single-end data, specify `--singleEnd` on the command line when you launch the pipeline. A normal glob pattern, enclosed in quotation marks, can then be used for `--reads`. For example: `--singleEnd --reads '*.fastq'`
+This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future.
 
-It is not possible to run a mixture of single-end and paired-end files in one run.
+## Main arguments
 
-### `--outdir`
-The output directory where the results will be saved.
+### `-profile`
```

Comment:
Shouldn't we talk about the profile in configs here as well?

Reply:
I don't understand what you mean..?

Reply:
in this whole paragraph about profile, we don't mention once the profile available through nf-core/configs.
I do think it should be mentionned

Reply:
I agree with @MaxUlysse but I think this is merged from the template so it should be introduced there instead?

Reply:
Aha! Yes you're right - that should definitely be mentioned 👍

Reply:
Updated

Reply:
Updated in https://github.com/nf-core/methylseq/pull/108/commits/cdf9ff33f2726b6ca6798536dd1d664db03019a1 for this pipeline and also in the template.